src/client/components/Nav.jsx

```diff
@@ -39,80 +39,82 @@ class Nav extends React.Component {
+         ${determinePanelStyles('/va-benefits', '/va-benefits/basic-information')}`}>
           <div>
-            <h5>Military Service</h5>
+            <h5>VA Benefits</h5>
             <ul className="usa-unstyled-list">
-              <li className={`${determineSectionStyles('/military-service/service-information')}`}>
-                Service
-              </li>
-              <li className={`${determineSectionStyles('/military-service/additional-information')}`}>
-                Additional Military
+              <li className={`${determineSectionStyles('/va-benefits/basic-information')}`}>
+                Basic Information
               </li>
             </ul>
           </div>
         </li>
         <li role="presentation" className={`four ${subnavStyles}
-         ${determinePanelStyles('/financial-assessment', '/financial-assessment/deductible-expenses')}`}>
+         ${determinePanelStyles('/household-information', '/household-information/deductible-expenses')}`}>
           <div>
-            <h5>Financial Assessment</h5>
+            <h5>Household Information</h5>
             <ul className="usa-unstyled-list">
-              <li className={`${determineSectionStyles('/financial-assessment/financial-disclosure')}`}>
+              <li className={`${determineSectionStyles('/household-information/financial-disclosure')}`}>
                 Financial Disclosure
               </li>
-              <li className={`${determineSectionStyles('/financial-assessment/spouse-information')}`}>
+              <li className={`${determineSectionStyles('/household-information/spouse-information')}`}>
                 Spouse
               </li>
-              <li className={`${determineSectionStyles('/financial-assessment/child-information')}`}>
+              <li className={`${determineSectionStyles('/household-information/child-information')}`}>
                 Child
               </li>
-              <li className={`${determineSectionStyles('/financial-assessment/annual-income')}`}>
+              <li className={`${determineSectionStyles('/household-information/annual-income')}`}>
                 Annual Income
               </li>
-              <li className={`${determineSectionStyles('/financial-assessment/deductible-expenses')}`}>
+              <li className={`${determineSectionStyles('/household-information/deductible-expenses')}`}>
                 Deductible Expenses
               </li>
             </ul>
           </div>
         </li>
-        <li role="presentation" className={`five last ${subnavStyles}
+        <li role="presentation" className={`five ${subnavStyles}
+         ${determinePanelStyles('/insurance-information', '/insurance-information/general')}`}>
+          <div>
+            <h5>Insurance Information</h5>
+            <ul className="usa-unstyled-list">
+              <li className={`${determineSectionStyles('/insurance-information/general')}`}>
+                General Insurance
+              </li>
```

Comment:
Seems like the panel would be too long if all of the questions appear in one view? I'm trying to run it locally to see it but getting errors.